components/tabs/stories/index.js

```diff
@@ -1,6 +1,8 @@
 export const argTypes = {
   items: { table: { disable: true }},
   selectorStyle: { table: { disable: true }},
+  style: { table: { disable: true }},
```

Comment:
Instead of style, can we use `customStyles` which is baked into the global storybook settings.